cogs/vxtwitconverter/vxtwitconverter.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import os
+
+from discord import Message, channel
+from urlextract import URLExtract
+
+from redbot.core import commands, data_manager
+from redbot.core.bot import Red
+
+
+class VxTwitConverter(commands.Cog):
+    """Converts twitter link embeds to vxtwitter"""
+
+    def __init__(self, bot: Red):
+        super().__init__()
+        self.bot = bot
+
+        # Initialize logger, and save to cog folder.
+        saveFolder = data_manager.cog_data_path(cog_instance=self)
+        self.logger = logging.getLogger("red.luicogs.vxtwitconverter")
+        if not self.logger.handlers:
+            logPath = os.path.join(saveFolder, "info.log")
+            handler = logging.FileHandler(filename=logPath, encoding="utf-8", mode="a")
+            handler.setFormatter(
+                logging.Formatter("%(asctime)s %(message)s", datefmt="[%d/%m/%Y %H:%M:%S]")
```

Comment:
@Injabie3 at one point we decided to do year/month/day, right?

```suggestion
                logging.Formatter("%(asctime)s %(message)s", datefmt="[%Y/%m/%d %H:%M:%S]")
```